### PR TITLE
add more quotes

### PR DESCRIPTION
**Pull request by aekong**

### CODE REVIEW

File: quotes.js [hunk 1]
```diff
@@ -59,4 +59,10 @@ module.exports = [
     "People who buy things are suckers.",
     "I'm going to get 12 eggs and part of a dead animal. Dealer's choice. Please and thank you.",
     "I wanna punch you in the face so bad right now."
+    "Normally, if given the choice between doing something and nothing, I’d choose to do nothing. But I will do something if it helps someone else do nothing. I’d work all night, if it meant nothing got done.",
+    "Dear frozen yogurt, you are the celery of desserts. Be ice cream or be nothing. Zero stars.",
+    "There has never been a sadness that can’t been cured by breakfast food.",
+    "If any of you need anything at all, too bad. Deal with your problems yourselves, like adults.",
+    "It’s pointless for a human to paint scenes of nature when they can go outside and stand in it.",
+
```
Comment (jamesseanwright) [inline quotes.js]: Please remove this blank line.

File: quotes.js [hunk 2]
```diff
@@ -59,4 +59,10 @@ module.exports = [
     "People who buy things are suckers.",
     "I'm going to get 12 eggs and part of a dead animal. Dealer's choice. Please and thank you.",
     "I wanna punch you in the face so bad right now."
```
Comment (jamesseanwright) [inline quotes.js]: You need to insert a comma at the end of line 61. This project was first deployed before Node.js supported trailing commas.

File: quotes.js (hunk 1 above)
Comment (aekong) [reply quotes.js]: done!

File: quotes.js (hunk 2 above)
Comment (aekong) [reply quotes.js]: removed trailing comma at the last array entry

Comment (aekong) [reply quotes.js]: and added comma after line 61 😄

Comment (jamesseanwright) [reply quotes.js]: Amazing! I'll get this merged!

**Comment by jamesseanwright** [comment]
@aekong I'm currently away from my main computer and I unfortunately haven't configured CI/CD for this, so I'll deploy it over the weekend.

**Comment by aekong** [comment]
no worries, thank you!

**Pull request merged**